dotcom-rendering/src/components/Bio.tsx

```diff
@@ -0,0 +1,71 @@
+import { css } from '@emotion/react';
+import { neutral, space, textSans14 } from '@guardian/source/foundations';
+import sanitise from 'sanitize-html';
+import { palette } from '../palette';
+
+/** Nesting is necessary in the bio styles because we receive a string of html from the
+ * field. This can contain the following tags:
+ * Blocks: p, ul, li
+ * Inline: strong, em, a
+ */
+const bioStyles = css`
+	${textSans14};
+	padding: ${space[1]}px 0;
+	color: ${palette('--mini-profiles-text-subdued')};
+	p {
+		margin-bottom: ${space[2]}px;
+	}
+	a {
+		color: ${palette('--link-kicker-text')};
+		text-underline-offset: 3px;
+	}
+	a:not(:hover) {
+		text-decoration-color: ${neutral[86]};
```

Comment:
Could this be added to the `palette`, and is there a dark mode version of this colour?